content/content-strategy/audit-content/analyse-evaluate/example.md

```diff
@@ -2,13 +2,12 @@
 layout: text/callout
 ---
 ### Example
-Use these tools for your content audit:
-  * website content analytics, such as Google Analytics
-  * user research feedback tools, such as surveys and behaviour tracking
-  * readability checkers
+Use these kinds of tools for your content audit:
```

Comment:
Would be good to apply the `(for example, x)` content pattern in the future maybe @annekemyers ?